File: 8.7.2025/Num Palindrome.java
import java.util.Scanner;
public class Main {
    public static void main(String[] args) {
        Scanner sc= new Scanner(System.in);
        System.out.println("Enter a number:");
        int n = sc.nextInt();
        int o =n;
        int rev = 0;
        int r;
        while(o!=0){
            r = o%10;
            rev = rev*10 + r;
            o = o/10;
        }
        if(rev == n){
            System.out.println("It's a Palindrome.");
        } else{
            System.out.println("It's not a Palindrome.");
        }
    }
}

File: 8.7.2025/Num Palindrome.c
#include <stdio.h>
int main() {
    int n,o,r,rev=0;
    printf("Enter a number:\n");
    scanf("%d",&n);
    o=n;
    while(o!=0){
        r = o%10;
        rev = rev*10 +r;
        o = o/10;
    }
    if(rev == n){
        printf("It's a Palindrome.");
    } else{
        printf("It's not a Palindrome.");
    }
}
